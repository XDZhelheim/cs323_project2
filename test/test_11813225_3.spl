int x(int a)
{
  return a;
}

int testtt()
{
  return x(5.0);
}