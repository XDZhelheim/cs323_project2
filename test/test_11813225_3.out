Error type 9 at Line 8: a functionâ€™s arguments mismatch the declared parameters
